### PR TITLE
Task/Set up /confirmation page for translations

### PR DESCRIPTION
**Pull request by pcraig3**
Make the confirmation page translate-able.

Very much like #45

### CODE REVIEW

File: views/confirmation/confirmation.pug [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
-    p However, if you still have further questions, there are several ways to get help.
+    p #{__('Now that your return is filed, you should be all set for the year.')}
+    p #{__('However, if you still have further questions, there are several ways to get help.')}
     ul
-      li Speak to someone at your nearest tax clinic
-      li Go online to #[a(href="https://apps.cra-arc.gc.ca/ebci/oecv/external/prot/cli_srch_01_ld.action" target="_blank") find a free tax clinic in your area]
-      li Speak to a CRA agent by calling 1-800-PLEASE-GO-AWAY
+      li #{__('Speak to someone at your nearest tax clinic')}
+      li #{__('Go online to')} <a href="https://apps.cra-arc.gc.ca/ebci/oecv/external/prot/cli_srch_01_ld.action" target="_blank">#{__('find a free tax clinic in your area')}</a>
+      li #{__('Speak to a CRA agent by calling 1-800-PLEASE-GO-AWAY')}
```
Comment (katedee) [inline views/confirmation/confirmation.pug]: 😂 can't believe I missed the PLEASE-GO-AWAY,
But also, we probably want this to be a tel link

Comment (pcraig3) [reply views/confirmation/confirmation.pug]: Yes, good catch.

Comment (pcraig3) [reply views/confirmation/confirmation.pug]: I will use your number from the offboarding page

Comment (nmakuch) [reply views/confirmation/confirmation.pug]: 1-800-PLEASE-GO-AWAY sounds so legit though

**Pull request merged**